### DISCUSSION TITLE
Struggling auth bug

### DISCUSSION THREAD

**Discussion by yeralexey** [jeremyephron/simplegmail#60] [Q&A]
Hi everyone!

_I posted the same message in another topic, not sure if it was correct place, so sorry for mistakes._

I am new in gihub, python and google api, so my question might be solved but i cant get how or where.

My script was running on VPS, so got in trouble that seems to be described here. Script suppose to run each 10 minutes, exactle in one week i got this:

**"Your browser has been opened to visit:

https://accounts.google.com/o/oauth2/auth?...........(my personal api oath data i guess).........&access_type=offline&response_type=code 

If your browser is on a different machine then exit and re-run this
application with the command-line parameter --noauth_local_webservernex
"**

So i stucked on this moment.... Can somebody explain me what shoud be done next? If i run it on my home ps - Authorisation window is being opened, i pass it but i cant do the same on my VPS.

Thanks, people.

**Comment by gexzam** [comment]
Hey @yeralexey, you have to run your script with this argument in command line: `--noauth_local_webserver`.
Then you have to copy that prompted link, paste in a visual browser and login into Gmail with the authorized account (remember when you created the json credentials file. I reffer to that gmail account). Then click in "Accept" or "Allow Access" (I can't remember button name) and there you go.
